qutebrowser/qt/machinery.py

```diff
@@ -280,6 +282,7 @@ def init(args: argparse.Namespace) -> SelectionInfo:
     info = _select_wrapper(args)
     if info.wrapper is not None:
         _set_globals(info)
+        log.init.debug(str(info))
```

Comment:
That's true.. I will try to move `log.init_log()` before `machinery.init()`.
IIRC this resulted in the crash the PR is supposed to fix, but I might have missed something.

Reply:
Wow, my bad. When trying to verify that my changes are not crashing I did always check with my system-wide qutebrowser instead of the local one..

I moved `init_log()` up in `early_init()` (8cd32c0ad).